hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/TestTags.java

```diff
@@ -22,6 +22,7 @@ private TestTags() {
         throw new UnsupportedOperationException("Utility class");
     }
 
+    public static final String ADHOC = "ADHOC";
```

Comment:
Probably should have a javadoc comment, like `ONLY_{EMBEDDED,REPEATABLE}` below, as it indicates "special handling required".

Reply:
Thanks @david-bakin-sl . This tag allows running one single suite on terminal